CONTRIBUTING.md

```diff
@@ -45,7 +45,7 @@ This section includes helpful troubleshooting hints to help you navigate the cod
     To do so: 
     1. `docker exec` into the test container and run `keytool -list -keystore $JRE_HOME/lib/security/cacerts -alias <IMPORTED_CERT_ALIAS>`. The alias of the certificate can be found in the test script.
         1. If import was successful, the certificate fingerprint (SHA) should be returned.
-    2. Navigate to `https://localhost:<PORT_OF_DAP/CONJUR_CONTAINER>` in your browser (make sure the DAP/Conjur container is still up and running). 
+    2. Navigate to `https://localhost:<PORT_OF_DAP/CONJUR_CONTAINER>` in your browser (make sure the Conjur Enterprise/Conjur container is still up and running). 
```

Comment:
Also change `<PORT_OF_DAP/CONJUR_CONTAINER>` to `<PORT_OF_CONJUR_CONTAINER>` here.